lib/rouge/lexers/powershell.rb

```diff
@@ -652,28 +97,47 @@ class Powershell < Shell
         rule %r/\s+/, Text
         rule %r/\$?"/, Str::Double, :double_quotes
         rule %r/\$'/, Str::Single, :ansi_string
-
         rule %r/'/, Str::Single, :single_quotes
-
         rule %r/\*/, Keyword
-
         rule %r/;/, Text
         rule %r/[^=\*\s{}()$"'`<]+/, Text
         rule %r/\d+(?= |\Z)/, Num
         rule %r/</, Text
         mixin :interp
       end
 
+      state :hashtable do 
+        rule %r/\w/, Name::Variable
+        rule %r/}/, Operator, :pop!
+        mixin :root
+      end
+
+      state :multiline do
+        rule %r/\.(#{MULTILINEKEYWORDS})/i, Keyword::Pseudo
+        rule %r/[\w,\d,\s,\.,\-,\,:\/,{,},<,>"*]/, Comment
+        rule %r/#>/, Comment, :pop!
+        mixin :root
```

Comment:
Same answer as well, I'm only creating a specific state here because most comments in most languages that i'm aware of don't include multi-line comments with magic powers, like the documentation-generating keywords. I'm also not convinced Keyword::Pseudo is correct, but I don't know what better class they can be put in. The terms don't really match what powershell calls them so I'm forced to make guesses as to what Rouge should call them.

Reply:
@aaroneg Looks like you missed the regex here..
`%r/[\w,\d,\s,\.,\-,\,:\/,{,},<,>"*]/` needs to be fixed here..